src/dashboard/infrastructure/integrations/site-kit.php

```diff
@@ -109,6 +120,24 @@ public function is_onboarded() {
 		return ( $this->is_site_kit_installed() && $this->is_setup_completed() && $this->is_connected() );
 	}
 
+	/**
+	 * Can user read site kit data.
+	 *
+	 * @param string $slug The slug of the data.
+	 * @return bool If the user can read the data.
+	 */
+	private function can_read_data( $slug ) {
+		$dashboard_sharing    = \get_option( 'googlesitekit_dashboard_sharing' );
+		$search_console_roles = isset( $dashboard_sharing[ $slug ] ) ? $dashboard_sharing[ $slug ]['sharedRoles'] : [];
+
+		// check if current user has one of those roles.
+		$current_user_has_access = \array_intersect( \wp_get_current_user()->roles, $search_console_roles );
+
+		$search_console_options = \get_option( 'googlesitekit_' . $slug . '_settings' );
+		$owner_id               = ( $search_console_options['ownerID'] ?? '' );
+		return $owner_id === \get_current_user_id() || $current_user_has_access;
```

Comment:
Hmm, I thought there was a PHP linting rule to require a newline above `return` statements. Guess I was mistaken.